tests/unit-tests/gconstruct/test_construct_graph.py

```diff
@@ -229,6 +229,28 @@ def check_feat_ops_noop():
     assert res[0].feat_name == feat_op1[0]["feature_col"]
     assert isinstance(res[0], Noop)
 
+def test_noop_string():
+    text_vector_data = {
```

Comment:
Can you also test the case when there is no separator?
Also the case when there is a separator and also truncate_dim.